dask/dataframe/io/parquet.py

```diff
@@ -121,7 +117,8 @@ def read_parquet(path, columns=None, filters=None, categories=None, index=None,
                         columns=[c for c in pf.columns if c in dtypes])
 
     for cat in categories:
```

Comment:
If they don't include the categories, the values get de-referenced upon load.
If they pass in a column name to categories which doesn't exist, it is ignored.
I'll fix None.